packages/desktop-client/src/components/transactions/TransactionsTable.jsx

```diff
@@ -367,7 +367,7 @@ const TransactionHeader = memo(
         />
         {showBalance && <Cell value="Balance" width={88} textAlign="right" />}
         {showCleared && <Field width={23} truncate={false} />}
-        <Cell value="" width={5 + scrollWidth ?? 0} />
+        <Cell value="" width={scrollWidth ? 5 + scrollWidth : 0} />
```

Comment:
This is the only visual change, (causing a visual test failure). The esbuild was just complaining that the existing change was a bug. (would never return 0). Should move this fix to a different PR to completely eliminate visual changes from this PR!

Reply:
Would `<Cell value="" width={5 + (scrollWidth ?? 0)} />` fix the warning and the visual regression?

Reply:
@joel-jeremy indeed you are correct, we don't want this to be 0, we want it to be at least 5, and plus the scrollWidth if available. Updated.